src/pages/VerifySettings/VerifySettings.tsx

```diff
@@ -36,7 +36,7 @@ export const preInstalledRequest = async (
   }
 
   try {
-    return await res.json();
+    return await res.json() as Promise<User>;
```

Comment:
Does this remove a Promise of a user or just a User as it's already awaited.

Reply:
removed `Promise`